test/k8s-integration/config/test-driver-windows.yaml

```diff
@@ -0,0 +1,20 @@
+StorageClass:
+  FromName: false
+  FromFile: sc-windows.yaml
```

Comment:
storageclass, snapshotclass, and capabilities should be templated as they are generated by the test runner, like https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/master/test/k8s-integration/config/test-config-template.in

Reply:
updated using template.

Reply:
Not seeing the update?

Reply:
will remove it.